ReactiveCocoa/ReactiveCocoa.h

```diff
@@ -62,8 +62,8 @@ FOUNDATION_EXPORT const unsigned char ReactiveCocoaVersionString[];
 #import <ReactiveCocoa/RACTuple.h>
 #import <ReactiveCocoa/RACUnit.h>
 
-#ifdef TARGET_OS_WATCH
-#elif __IPHONE_OS_VERSION_MIN_REQUIRED
+#if TARGET_OS_WATCH
```

Comment:
This is the same way as `TARGET_OS_MAC` of [this line](https://github.com/ReactiveCocoa/ReactiveCocoa/pull/2218/files#diff-8fad1d07697f66d8d17da19478832e4fR86).